python/mxnet/cython/ndarray.pyx

```diff
@@ -13,7 +13,7 @@ cdef class NDArrayBase:
     cdef int cwritable
 
     cdef _set_handle(self, handle):
-        cdef unsigned long ptr
+        cdef unsigned long long ptr
```

Comment:
The type is used to facilitate conversion in ctypes. the handle is stored in void*